substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/BundleSupport.java

```diff
@@ -111,6 +117,16 @@ final class BundleSupport {
     static final String BUNDLE_OPTION = "--bundle";
     static final String BUNDLE_FILE_EXTENSION = ".nib";
 
+    ContainerSupport containerSupport;
+    boolean useContainer;
```

Comment:
Do we really need a `useContainer` boolean. I'd prefer if we could define `containerSupport == null` as our `useContainer == true`.

Reply:
The problem is containerSupport might still exist while useContainer is false if it runs e.g. on Windows. Dockerfile and container tool can still be specified and inserted into the bundle if the OS is not Linux, but just the container creation and build inside the container is skipped.

If this behavior is not necessary (e.g. just abort the native-image build if it is the wrong OS), the useContainer flag can be removed